mithril.js

```diff
@@ -49,9 +49,9 @@ var m = (function app(window, undefined) {
 	 *
 	 */
 	function m(tag, pairs) {
-		for (var args = [], i = 1; i < arguments.length; i++) {
-			args[i - 1] = arguments[i];
-		}
+		// JSX compatibility with 'null' {pairs} argument
+		var args = Array.prototype.slice.call(arguments, (pairs === null ? 2 : 1));
```

Comment:
Nit: just set `i` to `pairs === null ? 2 : 1`. Otherwise, you're creating a perf regression. 

(Chrome and Firefox both don't usually like it when you pass the `arguments` object as an parameter. This is one case.)